requirement-generator/requirement-sources-jsons.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * The module that re-exposes the generated jsons with strict static types.
+ */
+
+import { readFileSync } from 'fs';
+import { Course, RequirementsJson } from './types';
+
+/** All courses fetch from Cornell Class API, with only the fields we need. */
+export const filteredAllCourses: { readonly [semester: string]: readonly Course[] } = JSON.parse(
+  readFileSync('requirement-generator/filtered-all-courses.json').toString()
```

Comment:
Same case here

Reply:
Haha, I did this for a different reason.
This file is *generated* by fetcher.ts, so a direct import will break linter and TS if you don't have the generated file in place. This `filtered-all-courses.json` file is now 40MB now. I'm debating and weighting whether we should check it into the repo.

Reply:
That makes sense got it